### DISCUSSION TITLE
Lab Assignment #5

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/22S-ASIAAM-191A#30] [Assignments]
## Due 5/4 before class 

Please use this thread to post your [lab assignment's #4](https://albertkun.github.io/22S-ASIAAM-191A/assignments/week5/lab_assignment/) link, which is the GitHub pages URL. [Click here for a reminder on how to get it](https://albertkun.github.io/22S-ASIAAM-191A/labs/week1/3/#using-github-pages).

### For example

https://albertkun.github.io/22s-asiaam-191a-sample/week5/index.html

**Comment by daikis12** [comment]
https://daikis12.github.io/22s-191a-assignments/week6/index.html

**Reply by albertkun** [reply]
Nice website! But the survey is broken :( 
![image](https://user-images.githubusercontent.com/8574425/167354505-224f969b-73b4-4786-b23c-aa4da57972aa.png)

Please check this link to fix it:

https://support.google.com/docs/answer/160166

**Reply by daikis12** [reply]
I believe it's fixed now!!

**Reply by albertkun** [reply]
Sadly, i dont think so: 
![image](https://user-images.githubusercontent.com/8574425/167357296-97ecf6f1-f053-44c7-93b8-127b0f30bab6.png)

**Reply by daikis12** [reply]
oh no :( 
what about now!?

**Reply by albertkun** [reply]
yup, it works, thanks!

**Comment by stefuntran** [comment]
https://stefuntran.github.io/22s-asiaam-191a-labs/week%206/index.html

**Reply by albertkun** [reply]
survey is not accessible 
![image](https://user-images.githubusercontent.com/8574425/167708105-5f2c46f6-3b30-472a-9e73-dc354e1cfbb4.png)

check this to fix it: https://support.google.com/docs/answer/160166

**Reply by stefuntran** [reply]
Whoops, forgot to change restrictions. It should work now hopefully.

**Comment by joonyloony** [comment]
https://joonyloony.github.io/22s-191a-assignments/week6/index.html

**Comment by ananyakodali17** [comment]
https://ananyakodali17.github.io/asian-am-191a-labs/week6/

**Comment by shubhgupta2018** [comment]
https://shubhgupta2018.github.io/22s-191a-assignments/week-6/index.html

**Comment by emmashanson** [comment]
https://emmashanson.github.io/22s-asianam-191a-labs/week6/index.html

**Comment by kris10fu** [comment]
https://kris10fu.github.io/22s-asiaam-191a-labs/week6/index.html

**Comment by willa-21** [comment]
https://willakurland.github.io/aa191-labs/Week-6.2/Week-6/index.html

**Comment by ntran27** [comment]
https://ntran27.github.io/22s-asiaam-191a-labs/week6/index.html

**Comment by XiongHer** [comment]
https://xiongher.github.io/22s-asiaam-191a-labs/week6/index.html

**Comment by kericsp** [comment]
https://kericsp.github.io/aa191-labs/week6/index.html

**Comment by ashley-lu** [comment]
https://ashley-lu.github.io/s22-asiaam-191a-assignments/week6lab/survey.html

**Comment by bittertang** [comment]
https://bittertang.github.io/22s-asiaam-191a-labs/week%206/

**Comment by antonyblin** [comment]
https://antonyblin.github.io/aas191a-assignments/week6/index.html

**Comment by hajovel** [comment]
https://hajovel.github.io/211S-191A-Assignments/week6/index.html

**Comment by codingdianat** [comment]
https://codingdianat.github.io/aa191-assignments/week%206/index.html

**Comment by spolcharoen** [comment]
https://spolcharoen.github.io/22s-asiaam-191a-labs/week6/index.html

**Comment by penelopelamb** [comment]
https://penelopelamb.github.io/22s-aas-191a-labs/week6/iceCream.html

**Comment by NishuKrish** [comment]
https://nishukrish.github.io/AsianAm191A-Nish/week6/index.html

**Comment by angelineeow** [comment]
https://angelineeow.github.io/22s-asiaam-191a-labs/week6/survey.html

**Comment by kaittli** [comment]
https://kaittli.github.io/22s-asiaam-191a-labs/week6/index.html

**Comment by lindseymardona** [comment]
https://lindseymardona.github.io/22s-asiaam-191a-labs/week6/index.html
